package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "dev": "NODE_PATH=./lib pm2 start lib/server.js --watch --interpreter babel-node --name pm2-app",
     "webpack": "webpack -wd",
-    "test": "jest"
+    "test": "jest --watch"
```

Comment:
This need to be remove in order to action & check can passed on github runner

Reply:
Done :+1: